test/scripts/manager/container-manager.spec.js

```diff
@@ -15,8 +15,33 @@ describe('container manager', function() {
             "templateURL": org.ekstep.contenteditor.config.pluginRepo + "/org.ekstep.textbookmeta-1.1/editor/collectiontree.html",
             "controllerURL": org.ekstep.contenteditor.config.pluginRepo + "/org.ekstep.textbookmeta-1.1/editor/collectionEditorApp.js"
         }
+        spyOn(org.ekstep.contenteditor.containerManager, 'loadNgModules').and.returnValue(Promise.resolve('Resolved'));
+
         var collectionManifest = org.ekstep.contenteditor.config.pluginRepo + "/org.ekstep.collectioneditor-1.0/manifest.json";
         org.ekstep.contenteditor.containerManager.register(container, collectionManifest);
         expect(org.ekstep.contenteditor.containerManager.register.calls.count()).toEqual(1);
     });
-});
\ No newline at end of file
+    it('when controller url is not defined', function(){
+        var container = {
+            "id": "tree",
+            "templateURL": org.ekstep.contenteditor.config.pluginRepo + "/org.ekstep.textbookmeta-1.1/editor/collectiontree.html",
```

Comment:
Refer this restcased for loadNgModules verification
https://github.com/ocombe/ocLazyLoad/blob/master/tests/unit/specs/ocLazyLoad.spec.js